tox.ini

```diff
@@ -0,0 +1,58 @@
+[tox]
+envlist = py36,py37,py38,py39,pypy3
+skip_missing_interpreters = True
+isolated_build = True
+minversion = 3.3.0
+
+[testenv]
+deps =
+    flake8~=3.7
+    flake8-bugbear
+    flake8-builtins~=1.4
+    flake8-import-order-jwodder
+    pytest~=6.0
+    pytest-cov~=2.0
+commands =
+    flake8 --config=tox.ini src test
+    pytest {posargs} test
+
+[pytest]
+addopts = --cov=pyfscacher --no-cov-on-fail
+filterwarnings = error
+norecursedirs = test/data
+
+[coverage:run]
+branch = True
+parallel = True
+
+[coverage:paths]
+source =
+    src
+    .tox/*/site-packages
+
+[coverage:report]
+precision = 2
+show_missing = True
+omit = src/pyfscacher/_version.py
+
+[flake8]
+application-import-names = pyfscacher
+doctests = True
+exclude = .*/,build/,dist/,test/data,venv/,_version.py
+hang-closing = False
+import-order-style = jwodder
+#max-doc-length = 80
+max-line-length = 80
+select = C,B,B902,B950,E,E242,F,I,W
+
+ignore =
+    B005,
+    E116, E121, E122, E126, E127, E128, E131, E133,
+    E221, E222, E225, E226, E227, E228, E231, E241, E251, E262, E265, E266,
+    E271, E272,
+    E302, E305, E306,
+    E402,
+    E501,
+    E721,
+    I201,
+    W503,
```

Comment:
why is it so many ignores?  it was copied from dandi-cli and there  we have only
```
[flake8]
max-line-length = 100
#ignore = D100,D101,D102,D103,D104,D105,D200,D201,D202,D204,D205,D208,D209,D210,D300,D301,D400,D401,D403,E24,E121,E123,E126,E226,E266,E402,E704,E731,F821,I100,I101,I201,N802,N803,N804,N806,W503,W504,W605
ignore = E203,W503
```
ie the long list is commented out and only two ignores are "active"

Reply:
I set this up using [my personal packaging boilerplate system](https://github.com/jwodder/pyrepo), so the ignores are to my personal preferences.  Since we'll be using black via pre-commit now, I should probably get rid of most of them.

Reply:
Ignores simplified.